attention-codelab/process_figshare.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
```

Comment:
Always add apache 2 license info to top. 

Add top level docstring/comment also to explain what this code does.

Reply:
Done. Please check if the license information is in the format you expected.

Reply:
I think you just need to swap the license and the comment.